src/AltinnCore/Runtime/RestControllers/ProcessController.cs

```diff
@@ -15,21 +23,38 @@ namespace AltinnCore.Runtime.RestControllers
     /// </summary>
     [Route("{org}/{app}/instances/{instanceOwnerId:int}/{instanceGuid:guid}/process")]
     [ApiController]
+    [Authorize]
     public class ProcessController : ControllerBase
```

Comment:
if we are to follow stylecop all public methods should come before private methods.

Reply:
ok